src/warzone/src/warzone/model/Player.java

```diff
@@ -204,7 +219,79 @@ public void issue_order() {
 			else {
 				GenericView.printWarning("Incorrect command or army number, please check the countryID and the number of army");
 			}			
-		} while (l_armyToIssue > 0 );		
+		} while (l_armyToIssue > 0 );	
+		
+		String[] l_commandArray;
+		
+		do {
+			
+			GenericView.println("Please perform an order or play a card. Type \"done\" when you have finished:");
+			
+			l_command = d_keyboard.nextLine();
+			l_commandArray = l_command.split("\\s+");
+			
+			switch(l_commandArray[0].toLowerCase()) {
+			
+				//advance countrynamefrom countynameto numarmies
+				case "advance" : { 
+
+					//TODO: Make sure the command is valid
+					
+					//this.d_orders.add(new AdvanceOrder(this, l_commandArray[1], l_commandArray[2], Integer.parseInt(l_commandArray[3]));
+					
+					break;
+				}
+				
+				//bomb countryID
+				case "bomb" : { 
+				
+					//TODO: Make sure the command is valid and player has the card. Remove the card from their list if they have it.
+					
+					//this.d_orders.add(new BombCardOrder(this, Integer.parseInt(l_commandArray[1])));
+					
+					break;
+				}
+				
+				//blockade countryID
+				case "blockade" : { 
+					
+					//TODO: Make sure the command is valid and player has the card. Remove the card from their list if they have it.
+					
+					//this.d_orders.add(new BlockadeCardOrder(this, Integer.parseInt(l_commandArray[1])));
+					
+					break;
+				}
+				
+				//airlift sourcecountryID targetcountryID numarmies
+				case "airlift" : {
+					
+					//TODO: Make sure the command is valid and player has the card. Remove the card from their list if they have it.
+					
+					//this.d_orders.add(new AirliftCardOrder(this, Integer.parseInt(l_commandArray[1]), Integer.parseInt(l_commandArray[2]), Integer.parseInt(l_commandArray[3])));
+					
+					break;
+				}
+				
+				//negotiate playerID
+				case "negotiate" : {
+					
+					//TODO: Make sure the command is valid and player has the card. Remove the card from their list if they have it.
+					
+					//this.d_orders.add(new NegotiateCardOrder(this, Integer.parseInt(l_commandArray[1])));
+					
+					break;
+				}
+				case "done" : {
+					break;
+				}
+				default : {
+					
+					GenericView.printError(l_commandArray[0].toLowerCase() + " is not a recognized order. Please enter a valid one.");
```

Comment:
1.put it into the function convertOrder(it was created), and other people will need to create a function for  each order convertion.
2.reduce the card after the card was issued .